tests/wicked/before_test.pm

```diff
@@ -99,7 +99,11 @@ sub run {
             assert_script_run('make ; make install', timeout => 600);
         } elsif (my $wicked_repo = get_var('WICKED_REPO')) {
             record_info('REPO', $wicked_repo);
-            ensure_ca_certificates_suse_installed() if ($wicked_repo =~ /suse\.de/);
+            if ($wicked_repo =~ /suse\.de/ && script_run('rpm -qi ca-certificates-suse') == 1) {
+                my $version = generate_version('_');
+                zypper_call("ar --refresh http://download.suse.de/ibs/SUSE:/CA/$version/SUSE:CA.repo");
+                zypper_call("in ca-certificates-suse");
+            }
```

Comment:
take a look `ensure_ca_certificates_suse_installed` https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/bc1b29ef32/lib/utils.pm#L1828

Reply:
because of this https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12109 I do not use it